app/src/main/java/no/usn/mob3000/data/model/content/NewsDto.kt

```diff
@@ -21,7 +21,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class NewsDto(
     @SerialName("id")
-    val newsId: String,
+    val newsId: String? = null,
```

Comment:
News items will always have an ID. This change makes the tabulated data more difficult to work with and needs to be reverted.